dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -1029,10 +1029,10 @@ do
 done
 
 # Check which old packages will be removed (you need Python 3.8+ and dev/requirements.txt installed)
-python ${AIRFLOW_REPO_ROOT}/dev/provider_packages/remove_old_releases.py --directory .
+breeze release-management clean-old-provider-artifacts --directory .
```

Comment:
Now that we move to breeze command I believe the comment above of python 3.8 + dev/requirements.txt is no longer relevant ?

BTW I am not sure what i the output of this command?
For the script it was outputed a long list which was very hard to understand. If we can have friendlier output that is easy to read (maybe breakdown by provider name rather than a long mixed list) that can be very helpful!

Reply:
Makes sense around removal of execute flag. I will make the changes. In terms of beautifying, shall we land the major change first and then follow up PR for it?